### DISCUSSION TITLE
Translate Documentation

### DISCUSSION THREAD

**Discussion by cesarhoeflich** [pnp/pnpcore#537] [Q&A]
I'm wanting to translate the documentation to **pt-br**, however, when I downloaded the project and tried to use DocFx's multi-language model, the model used doesn't have the DROPDOWN of language choice.

How should I proceed in this case?

I can perform the translation by inserting a **/pt-br/** subfolder so that when I accessed the https://pnp.github.io/pnpcore/pt-br link, it opens the translated doc.

Awaiting return.

**Comment by jansenbe** [comment]
That's a great idea @cesarhoeflich ! Adding @pkbullock for ideas on updating our docfx template as he created the version we're using. Do you happen to have a sample repo showing a docfx template with a language picker?
